google/cloud/bigtable/row.h

```diff
@@ -33,18 +34,19 @@ inline namespace BIGTABLE_CLIENT_NS {
 class Row {
  public:
   /// Create a row from a list of cells.
-  Row(std::string row_key, std::vector<Cell> cells)
+  template <typename T>
+  Row(T row_key, std::vector<Cell> cells)
```

Comment:
I think this should also be `T&&` and `std::forward<T>(row_key)`

Reply:
Fixed.